hw/ip/otbn/dv/uvm/sva/otbn_sva.core

```diff
@@ -11,6 +11,7 @@ filesets:
       - lowrisc:fpv:csr_assert_gen
```

Comment:
Add a `- lowrisc:prim:assert`  please, as well as a dependency on `lowrisc:ip:otbn` for the otbn_reg_pkg (looks like this one was missed before).

Reply:
Oops, good catch, thanks!